source/instructors/teaching/addstudents.rst

```diff
@@ -4,4 +4,6 @@
 Inviting students to your course
 ================================
 
-See :ref:`Add or Remove Students in a Course <add-remove-students>`
\ No newline at end of file
+See :ref:`Add or Remove Students in a Course <add-remove-students>`
+
+.. Note:: When integrating with an external system, you do not :ref:`add/invite <add-remove-students>` students or teachers to your course. All is handled when the users access an assignment in the system mapped to Codio and the features to invite students into the course are not enabled.
```

Comment:
When integrating with an external system, you do not need to :ref:`add/invite <add-remove-students>` students or teachers to your course. When users access an assignment from the system mapped to Codio, they will automatically be added to the Codio course. The features to invite students into a Codio course directly are not enabled when your course is connected to an external system such as an LMS.